packages/app/components/Today/QueueCheckInButton.tsx

```diff
@@ -60,45 +54,30 @@ export default function TodayPageCheckinButton(): ReactElement {
             }
           }}
         >
-          <h3>Which queue would you like to check into?</h3>
-          <Radio.Group
-            value={queueToCheckInto}
-            onChange={(e) => setQueueToCheckInto(e.target.value)}
+          <h3>Begin searching for a room</h3>
+          <Select
```

Comment:
I couldn't test it on staging (it ded) but wondering if the dropdown would overflow the  modal or just like hide

Reply:
it has a max size and it is scrollable within that window